shimmingtoolbox/optimizer/lsq_optimizer.py

```diff
@@ -208,6 +216,32 @@ def _initial_guess_zeros(self):
 
         return current_0
 
+    def _residuals_mse_jacobian(self, coef, unshimmed_vec, coil_mat, factor):
+        """ Jacobian of the function that we want to minimize
+        The function to minimize is :
+        np.mean((unshimmed_vec + np.sum(coil_mat * coef, axis=1, keepdims=False)) ** 2) / factor+\
+           (self.reg_factor * np.mean(np.abs(coef) / self.reg_factor_channel))
+        If you want to see the mathematical expression for the jacobian, of this function or for the mae one,
+        please contact : aurelienpujolmpro@gmail.com
```

Comment:
Write it on GitHub so we don't need to email you in the future please.

Reply:
👉  https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions